### DISCUSSION TITLE
Nacht 2.0

### DISCUSSION THREAD

**Discussion by ImTander** [nzp-team/nzportable#688] [Map Releases]
Nacht der Untoten 2.0! 
What is it? Simple really, just Nacht with perks & PaP!

This is just built on the standard Nacht that comes with the game, so credit for the map still really goes out to the NZ:P Team, I didn't do much.

**What's new?**
- the four core perks 
- three hidden teddies to unlock PaP

![nach2jug](https://github.com/nzp-team/nzportable/assets/161973907/e7352d37-d94d-4a18-a43b-55930a7fc25f)
![Nacht2](https://github.com/nzp-team/nzportable/assets/161973907/4ae71d36-d981-411b-a4d4-956d326cb968)
![nacht2pap](https://github.com/nzp-team/nzportable/assets/161973907/fd6dc884-a62d-479c-91d5-9d7fa150fbb6)
![nacht2qr](https://github.com/nzp-team/nzportable/assets/161973907/ad86dc44-2c58-4040-9093-2e7b224cf4eb)

PLEASE NOTE : this also uses the original Nacht waypoints, so if you had any issues/made any adjustments to those you'll also want to do that here. Or, you can simply copy your org Nacht .way file, paste, and rename.

Good Luck!

# [DOWNLOAD](https://archive.org/download/nacht-2.0_202403/Nacht%202.0.zip)

**Comment by ToonLink0001** [comment]
nice map

**Comment by MotoLegacy** [comment]
Cool take. Pro tip -- we bundle a version of the Double Tap 1.0 machine model in the game files, so you should update your Double Tap entity to use that model instead :) `models/machines/hl_scale/double_tap.mdl`

**Reply by ImTander** [reply]
This has been updated, thanks for letting me know!

![dtupdate](https://github.com/nzp-team/nzportable/assets/161973907/316490a8-ee5e-458d-9892-2c590c33fe33)

**Comment by Conmiester** [comment]
This reminds me of the vibes of some Call of Duty: World At War zombie mods that adds the core 4 perks and Pack a Punch for Nacht Der Untoten, It's nice to see that take in NZ:P's Nacht Der Untoten map.

**Comment by PHDFLOPPERGUY56** [comment]
Would love to see a version that adds the Black Ops 1 perks as well!

**Comment by spectrum1-wave** [comment]


**Reply by PrimiseRichtofen** [reply]
okay so there's one in spawn behind the stairs. there's one on a crate wheere grenade wall buy is and one next to shelf where radio is/

**Comment by ageofdecimation** [comment]
how umlock pap

**Reply by nonMat** [reply]
What's new?

    the four core perks
    three hidden teddies to unlock PaP

AAs to where thte teddys are idk

**Reply by ageofdecimation** [reply]
found em

**Comment by spectrum1-wave** [comment]
Ermmmmm.....
<img width="1366" height="768" alt="Screenshot 2025-12-22 12 52 27 PM" src="https://github.com/user-attachments/assets/fee8c107-4da8-427a-8ef3-bd34fd288278" />

**Reply by spectrum1-wave** [reply]
you can just stand by the stairs and they pile up
